v4/source/teams.yaml

```diff
@@ -970,6 +1294,17 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+      x-code-samples:
+        - lang: 'Go'
+          source: |
+            import "github.com/mattermost/platform/model"
+
+            Client := model.NewAPIv4Client("https://your-mattermost-url.com")
+            Client.Login("email@domain.com", "Password1")
+
+            teamID := "zWEyrTZ7GZ22aBSfoX60iWryTY"
+
+            _, resp := Client.InviteUsersToTeam(teamID, []string{"test@domain.com", "test2@domain.com"})
```

Comment:
Why do you ignore the return value here?

Reply:
I do not ignore it anymore